DFC.ServiceTaxonomy.GraphSync/Notifications/GraphSyncNotifier.cs

```diff
@@ -145,6 +150,10 @@ private string GetContentTypeDisplayName(ContentItem contentItem)
                 technicalMessage,
                 exceptionText);
 
+            //publish to slack
+            string slackMessage = BuildSlackMessage(traceId, userMessage, technicalMessage, exception);
+            await _slackMessagePublisher.SendMessageAsync(slackMessage);
```

Comment:
Can we swallow any exceptions sending the message, so that that the user still gets to see the original notification if there's an issue with slack?